src/tenant/models.py

```diff
@@ -26,6 +29,10 @@ def check_tenant_name(name):
         raise ValidationError("Invalid string used for the tenant name.")
 
 
+def default_trial_end_date():
+    return date.today() + timedelta(days=60)
```

Comment:
Need to use Django's timezone.now() so that our TZ info is taken into account.  BUUUUT Considering Semester.last_day doesn't even do that, I'll make it a new issue.